src/web/client/dal/fileSystemProvider.ts

```diff
@@ -242,11 +244,29 @@ export class PortalsFS implements vscode.FileSystemProvider {
     }
 
     async rename(): Promise<void> {
-        throw new Error("Method not implemented.");
+        oneDSLoggerWrapper.getLogger().traceError(
+            telemetryEventNames.WEB_EXTENSION_RENAME_NOT_SUPPORTED,
+            WEB_EXTENSION_METHOD_NOT_IMPLEMENTED,
+            { name: telemetryEventNames.WEB_EXTENSION_RENAME_NOT_SUPPORTED, message: WEB_EXTENSION_METHOD_NOT_IMPLEMENTED} as Error,
```

Comment:
why is this relogging of eventName neded here?

Reply:
as per conversation with @BidishaMS this telemetry logging in oneDSLogger is not required hence reverting the changes.